libroll.py

```diff
@@ -108,3 +108,27 @@ def is_well_known_term():
 
 
 ####################################################################################################
+
+
+def read_json_file(file_path: str) -> dict:
+    """
+    Reads a JSON file and returns the parsed contents.
+    """
+    import json
+    with open(file_path, 'r') as file:
+        return json.load(file)
+
+
+####################################################################################################
+
+
+def write_json_file(file_path: str, data: dict):
+    """
+    Writes a JSON file with the given data.
+    """
+    import json
```

Comment:
Is this kind of imports in function (esp. if repeated in multiple functions) a convention in Python?

Is the net effect that `json` is only loaded if you use these functions, but not loaded otherwise (which it would be if we imported at the top of the file)?

I'm guessing there is a mechanism throught which something that has already been imported is not reloaded again, right?